fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsManager.java

```diff
@@ -86,12 +87,12 @@ public void alterTableStatistics(AlterTableStatsStmt stmt) throws AnalysisExcept
      * @throws AnalysisException if table, column or partition not exist
      */
     public void alterColumnStatistics(AlterColumnStatsStmt stmt) throws AnalysisException {
-        Table table = validateTableName(stmt.getTableName());
+        OlapTable table = (OlapTable) validateTableName(stmt.getTableName());
```

Comment:
we should do instanceof and cast when we do partition check, since not only OlapTable has statistics